bug_update_page.php

```diff
@@ -769,7 +770,7 @@
 if( $t_bottom_buttons_enabled ) {
 ?>
 	<div class="widget-toolbox padding-8 clearfix">
-		<input <?php helper_get_tab_index(); ?>
+		<input <?php echo helper_get_tab_index(); ?>
```

Comment:
The same change is needed at the beginning of the file if `$t_top_buttons_enabled`